src/controllers/activegate/reconciler/statefulset/statefulset.go

```diff
@@ -47,6 +48,7 @@ type statefulSetProperties struct {
 	initContainersTemplates []corev1.Container
 	containerVolumeMounts   []corev1.VolumeMount
 	volumes                 []corev1.Volume
+	log                     logr.Logger
```

Comment:
is this necessary ?

Reply:
isn't there a `log` on the package level ?

Reply:
Yes, at this point it's no longer necessary. These changes were made long before the per-package loggers were introduced.